tests/integration/client_setup_js.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tests
+
+import (
+	"github.com/sourcenetwork/defradb/node"
+	"github.com/sourcenetwork/defradb/tests/clients"
+	"github.com/sourcenetwork/defradb/tests/clients/js"
+)
+
+func init() {
+	goClient = false
+	httpClient = false
+	cliClient = false
+	jsClient = true
+	skipNetworkTests = true
+	skipBackupTests = true
```

Comment:
todo: Please document why net and backup are skipped (within this func)

Reply:
done. Let me know if its a good enough explanation.

Reply:
I agree it could be useful. I'd prefer to do this in a separate PR since this one is already large. It will likely require changes to the backup API as well.

Reply:
Yes, completely on onboard with that, and it is a different priority really to this main body of work.